documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -22,7 +22,7 @@ Upgrading Strimzi from the previous minor version to version {ProductVersion}.
 Multi-version::
```

Comment:
`{ProductVersion}` renders as "latest", which means we're talking about "version latest" everywhere. It's not really clear whether this is a named version called "latest", bad English, or a mistake in how the documentation is generated.

Reply:
That is expected in main builds. IT should be set to the release version in release builds.

Reply:
Yeah, it's fine in the downloaded docs, but not on https://strimzi.io/docs/operators/latest/deploying.html#assembly-upgrade-str. Is that unfixable?

Reply:
You mean this?

![Screenshot 2021-11-15 at 14 52 53](https://user-images.githubusercontent.com/5658439/141793530-67c329f7-3d46-4325-97dc-725ebab4feb5.png)

Reply:
fYI: It looks like the docs are badly rendered or there is some bug. I will have a look at it.

Reply:
@tombentley I opened strimzi/strimzi.github.io#264 for it.